### PR TITLE
Fixed encoding & URL issues for users & groups pages

### PR DESCRIPTION
**Pull request by bexsoft**
fixes https://github.com/minio/console/issues/752

## What does this do?

Fixes an issue with encoding & users

## How does it look?

Screen Shot 2021-07-23 at 18 35 13
Screen Shot 2021-07-23 at 18 34 10

### CODE REVIEW

File: portal-ui/src/screens/Console/Users/ListUsers.tsx [hunk 1]
```diff
@@ -155,7 +155,8 @@ const ListUsers = ({ classes, setErrorSnackMessage }: IUsersProps) => {
   };
 
   const viewAction = (selectionElement: any): void => {
-    history.push(`/users/${selectionElement.accessKey}`);
+    console.log(encodeURI(selectionElement.accessKey));
```
Comment (dvaldivia) [inline portal-ui/src/screens/Console/Users/ListUsers.tsx:158]: remove

File: swagger-console.yml [hunk 2]
```diff
@@ -1269,13 +1269,13 @@ paths:
       tags:
         - AdminAPI
 
-  /users/{name}/groups:
+  /user/groups:
```
Comment (dvaldivia) [inline swagger-console.yml:1272]: I think we should keep `{name}` in the URL, as this is more RESTful, for the resource `{user}` put this or get that

**Review by dvaldivia — CHANGES_REQUESTED** [review]

File: swagger-console.yml (hunk 2 above)
Comment (bexsoft) [reply swagger-console.yml:1272]: This is causing issues with names that contain slashes, that is why I changed it to be in query instead. Also I changed to be singular to handle this

Comment (dvaldivia) [reply swagger-console.yml:1272]: ok, that makes sense then

**Review by dvaldivia — APPROVED** [review]

File: portal-ui/src/screens/Console/Users/ListUsers.tsx (hunk 1 above)
Comment (bexsoft) [inline portal-ui/src/screens/Console/Users/ListUsers.tsx:158]:
```suggestion
    
```

**Review by Alevsk — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**